### DISCUSSION TITLE
IFAK doesn't have disinfectant. Should it?

### DISCUSSION THREAD

**Discussion by Comrade-Soupbeans** [CleverRaven/Cataclysm-DDA#46737] [Q&A] [answered]
Military IFAKs don't have an kind of disinfectant in them. Is that intentional? I feel like they should since the civilian First aid kits do or, if they don't there should be a larger medic / Corpsman kit that is slightly rarer but has most supplies.

**Comment by anothersimulacrum** [comment, marked as answer]
https://www.military.com/equipment/individual-first-aid-kit-ifak
> 1 Utility Pouch 
> 1 Tourniquet 
> 1 Elastic bandage kit
> 1 Bandage GA4-1/2” 100’s
> 1 Surgical adhesive tape
> 1 Nasopharyngeal airway kit
> 4 Surgical gloves
> 1 Combat gauze dressing

It should contain this, so no, it should not have antiseptic/disinfectant.

**Comment by actual-nh** [comment]
There are indeed backpack-sized items for combat medics (aka "corpsmen" for at least the Navy and Marines), which might well contain antiseptic/disinfectant; these appear to vary by service and - to a likely variable degree of discretion - the medic/corpsman in question (see [this abstract](https://pubmed.ncbi.nlm.nih.gov/32203608/), for instance).

**Reply by Comrade-Soupbeans** [reply]
is there an item ingame? i can't seem to find it via debug and i've never found it in military loot tables in my time playing or on zombie soldiers

**Reply by actual-nh** [reply]
I can't find one. (BTW, the IFAK is no longer actually an item, technically, but an "item group" - pouch plus other stuff, which can be either a full one or a used one. But I looked in both `items` and `itemgroups`.)

**Reply by BernardKD** [reply]
wath's IFAK ?

**Reply by Jan-Blasiak** [reply]
Infantry
First
Aid 
Kit

**Comment by Jan-Blasiak** [comment]
As a paranoiac IRL I have personally augmented my IFAK to have everything I may need in case of injury, indigestion or headache. The augmented IFAK could be added as a rare item group to prepper spawns.
